ops/charm.py

```diff
@@ -808,6 +808,10 @@ class PebbleCustomNoticeEvent(PebbleNoticeEvent):
     """Event triggered when a Pebble notice of type "custom" is created or repeats."""
 
 
+class PebbleChangeUpdatedEvent(PebbleNoticeEvent):
```

Comment:
TODO benhoyt: should we have a `.change` attribute or `.get_change()` method on this class, that looks up the associated change via the notice key (= change ID)?

Reply:
> Do you think we should have a `.change` attribute or `.get_change()` method on this class, that looks up the associated change via the notice key (= change ID)?
> 
> A helper might be nice, though as it is it's not hard to get:
> 
> ```python
> change = event.workload.pebble.get_change(event.notice.key)
>     # vs
> change = event.get_change()
> ```

I think a helper would be nice. It seems like if you're observing change events then you're probably going to want to get hold of the actual change, so it makes sense to have it 'at hand' at the top level of the event namespace.

> I'd probably make it `get_change()` rather than a property, because it has to perform a call to Pebble.

:100:

Reply:
Added.